db/migrate/20200427200605_create_job_postings.rb

```diff
@@ -0,0 +1,16 @@
+class CreateJobPostings < ActiveRecord::Migration[5.2]
+  def change
+    create_table :job_postings do |t|
+      t.string :title
+      t.integer :salary
+      t.text :description
+      t.integer :number_positions
+      t.string :keywords
```

Comment:
Should the keywords be an array? Or are you planning to separate them with commas?

Reply:
I was planning to separate them with commas.